README.adoc

```diff
@@ -75,6 +75,13 @@ Once you're done playing with riff (see samples below), you can destroy everythi
 make teardown
 ----
 
+To unset the environment variables for building images with docker in minikube, issue:
```

Comment:
This could read like "to build in minikube, some vars need to be unset, here's how".

Should we use "To unset the environment variables *used* for building images ..."

Reply:
Yes we should. I'll fix that.

Reply:
@ericbottard Fixed.